### PR TITLE
added expiration note ( issue #253 )

### PR DESCRIPTION
**Pull request by saimaheshtaduri**
I added the expiration notes and a tooltip as mentioned. 

please tell me if you need any more changes. 
Thanks :)

### CODE REVIEW

File: packages/web/src/pages/match/single-match.tsx [hunk 1]
```diff
@@ -93,6 +94,12 @@ const SingleMatch: React.FC = () => {
       </div>
     );
   } else {
+
+    let startgamedate = new Date(matchData.startgametime * 1000); 
+    let currentday = new Date();
+    let days = config.matchAreStoredForDays - differenceInDays(currentday, startgamedate);
+    let expiryDate = new Date(new Date().setDate(currentday.getDate() + days)).toLocaleDateString('en-us', { weekday:"long", year:"numeric", month:"short", day:"numeric"});
```
Comment (petrvecera) [inline packages/web/src/pages/match/single-match.tsx:101]: I think that all these variables can be `const`. It's better to use it https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

We can let this to be empty. 
```
toLocaleDateString('en-us', { weekday:"long", year:"numeric", month:"short", day:"numeric"})
```
We are using dates on more parts of the app and it's always formatted for the user - depending on where they live :)

**Review by petrvecera — CHANGES_REQUESTED** [review]
Awesome. Looks great! Left just a little comment. And if you could just run `yarn fix` before commit. So it formats the file :) 

Otherwise it's exactly as we needed.

File: packages/web/src/pages/match/single-match.tsx (hunk 1 above)
Comment (saimaheshtaduri) [reply packages/web/src/pages/match/single-match.tsx:101]: Thanks @petrvecera 
I will update them now. 
Thanks :)

Comment (saimaheshtaduri) [reply packages/web/src/pages/match/single-match.tsx]: @petrvecera  done. please review it now

**Comment by petrvecera** [comment]
Awesome thank you 👍

**Pull request merged**